cmd/mv-main.go

```diff
@@ -243,17 +246,32 @@ func mainMove(cliCtx *cli.Context) error {
 			fatalIf(err.Trace(), "Unable to parse the provided url.")
 		}
 		if _, ok := client.(*S3Client); ok {
-			enabled, err := isBucketLockEnabled(ctx, urlStr)
-			if err != nil {
-				fatalIf(err.Trace(), "Unable to get bucket lock configuration of `%s`", urlStr)
+			var lifeCycleConfig *lifecycle.Configuration
+			var legalHoldStatus minio.LegalHoldStatus
+			var retentionMode minio.RetentionMode
+			var err *probe.Error
+			ignoreErr := "The specified object does not have a ObjectLock configuration"
+			if lifeCycleConfig, err = client.GetLifecycle(ctx); err != nil && strings.Contains(err.Cause.Error(), ignoreErr) {
```

Comment:
There is no such thing called ILM lock, I don't think we need to call this function.

Reply:
@vadmeste 
Done. Removed it.